frontend/src/main/scala/bloop/dap/BloopDebuggeeRunner.scala

```diff
@@ -55,76 +71,57 @@ private final class MainClassDebugAdapter(
       mainClass.environmentVariables,
       RunMode.Debug
     )
-
     runState.map(_.status)
   }
 }
 
 private final class TestSuiteDebugAdapter(
     projects: Seq[Project],
     filters: List[String],
-    state: State
-) extends DebuggeeRunner {
-  private lazy val allAnalysis = state.results.allAnalysis
-  def classFilesMappedTo(
-      origin: Path,
-      lines: Array[Int],
-      columns: Array[Int]
-  ): List[Path] = {
-    DebuggeeRunner.classFilesMappedTo(origin, lines, columns, allAnalysis)
+    initialState: State,
+    ioScheduler: Scheduler
+) extends BloopDebuggeeRunner(initialState, ioScheduler) {
+  override def name: String = {
```

Comment:
What is the name needed for?

Reply:
The name is used by the `DebugServer` for logging when there is a problem:
- a timeout waiting for the `DebuggeeRunner` to start
- a timeout waiting for the `Terminated` event